{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/settings.py

```diff
@@ -211,36 +188,47 @@
 if DEBUG:  # for testing
     REST_FRAMEWORK["DEFAULT_AUTHENTICATION_CLASSES"].append("rest_framework.authentication.SessionAuthentication")
     REST_FRAMEWORK["DEFAULT_RENDERER_CLASSES"].append("rest_framework.renderers.BrowsableAPIRenderer")
-#
-# Static files (CSS, JavaScript, Images)
-#
-{% if cookiecutter.client_app != "React" -%}
+
+# DIRS Definition
+
 STATIC_ROOT = os.path.join(BASE_DIR, "static")
 MEDIA_ROOT = os.path.join(BASE_DIR, "media-files")
-{% else %}
-STATIC_ROOT = os.path.join(BASE_DIR, "..", "client", "build")
-MEDIA_ROOT = os.path.join(BASE_DIR, "..", "client", "build", "static")
-{% endif -%}
-
 STATIC_URL = "/static/"
 MEDIA_URL = "/media/"
 
-{%- if cookiecutter.client_app != 'None' %}
-{%- if cookiecutter.client_app == 'Vue3' %}
-STATICFILES_DIRS = [
-    os.path.join(BASE_DIR, "../client/dist/static"),
-]
+# CLIENT SIDE BUILD DIRS
+
+{% if cookiecutter.client_app == "Vue3" %}
+CLIENT_BUILD_DIR = "../client/dist/"
+CLIENT_STATIC_FILES = "../client/dist/static"
 {% elif cookiecutter.client_app == 'React' %}
-STATICFILES_DIRS = [
-    os.path.join(STATIC_ROOT, "static"),
-]
-{% endif -%}
-{% endif -%}
+CLIENT_STATIC_FILES = "../build/static"
+CLIENT_BUILD_DIR = "../client/dist/"
+{% else %}
+CLIENT_STATIC_FILES = STATIC_ROOT
+CLIENT_BUILD_DIR = ""
+{% endif %}
 
+TEMPLATES = [
+    {
+        "BACKEND": "django.template.backends.django.DjangoTemplates",
+        "DIRS": [
+            CLIENT_BUILD_DIR
+        ],
+        "APP_DIRS": True,  # this setting must come after "DIRS"!
+        "OPTIONS": {
+            "context_processors": [
+                "django.template.context_processors.debug",
+                "django.template.context_processors.request",
+                "django.contrib.auth.context_processors.auth",
+                "django.contrib.messages.context_processors.messages",
+            ]
+        },
+    }
+]
 
-STATICFILES_FINDERS = [
-    "django.contrib.staticfiles.finders.FileSystemFinder",
-    "django.contrib.staticfiles.finders.AppDirectoriesFinder",
```

Comment:
^ This change might be unexpected to people familiar with Django. I know I'm accustomed to being able to add static files in the app directories and have those auto-detected. I don't feel strongly about this, but is there a reason that this _should_ be removed?